pages/login.js

```diff
@@ -1,5 +1,24 @@
 // TODO(alishaevn): redo this page when the time comes
+import { useSession, signIn, signOut } from 'next-auth/react'
 
-const Login = () => <h1>This is the login page.</h1>
+const Login = () => {
+  const { data: session } = useSession()
+  console.log({session})
+  if (session) {
+    return (
+      <div className='container'>
+        Signed in as {session?.user?.email} <br />
```

Comment:
Minor non-blocker; I think the `session?.` could be `session.`; we're already guarding that we have a session.  (Unless I'm misreading the `?.` operator).

Also, if this evaluates as `null` (we don't have a user email which is feasible), the text will say "Signed in as", a minor inconsistency in language.  Again, a non-blocker.

Reply:
agreed- after leaving my comment above, made the realization that this page will be going away anyways in this ticket: 

https://github.com/scientist-softserv/webstore-component-library/issues/128